modules/blast.nf

```diff
@@ -67,6 +76,26 @@ process blastn {
         mv ${sample_id}_${db_id}_blast.csv ${sample_id}_${db_id}_blast_tmp.csv
         add_db_metadata.py -m ${db_dir}/metadata.json -b ${sample_id}_${db_id}_blast_tmp.csv -d ${db_id} > ${sample_id}_${db_id}_blast.csv
     fi
+
+    cat <<-EOL_VERSIONS > ${sample_id}_${db_id}_blastn_provenance.yml
+    - process_name: "${task.process}-${db_id}"
+      tools:
+      - tool_name: blastn
+        tool_version: \$(blastn -version | head -n1 | sed 's/blastn: //g')
+        parameters:
+        - parameter: "perc_identity"
+          value: ${params.minid}
+        - parameter: "qcov_hsp_perc"
+          value: ${params.mincov}
+          database used: ${db_name}
```

Comment:
That looks good. I think it might make sense to just have an optional `databases` section within a `process`. I've posted an issue on our provenance schema repo here:

https://github.com/BCCDC-PHL/pipeline-provenance-schema/issues/3

Reply:
For the one relevant process (`blastn`), I included an optional `databases` list.